### PR TITLE
Added recipe for oslo.serialization

### PR DESCRIPTION
**Pull request by pmlandwehr**
Not sure if @anguslees wants in on this one too...

### CODE REVIEW

**Comment by anguslees** [comment]
@pmlandwehr: sure, why not.  I'm passingly familiar with most of the openstack libraries, and at this point I'm already in so deep that automation is my only hope.

**Comment by patricksnape** [comment]
```
conda.resolve.Unsatisfiable: The following specifications were found to be in conflict:
  - oslo.serialization 2.13.0 py34_0
  - python 3.4*
```

**Comment by pmlandwehr** [comment]
that's what i get for building locally...

**Comment by pmlandwehr** [comment]
Hmm. Builds fine on 3.4 here. I'm thinking this is a conda issue. Closing/reopening and hoping...

**Comment by pmlandwehr** [comment]
One more try...

File: recipes/oslo.serialization/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - pbr >=1.6
```
Comment (jakirkham) [inline recipes/oslo.serialization/meta.yaml]: `pbr` is needed at build time?

**Comment by jakirkham** [comment]
Could you please merge this with `master` @pmlandwehr ? I expect that CircleCI is just building with some out-of-date code, which has since been fixed.

File: recipes/oslo.serialization/meta.yaml (hunk 1 above)
Comment (anguslees) [inline recipes/oslo.serialization/meta.yaml]: See the discussion on https://github.com/conda-forge/staged-recipes/pull/1575

**Comment by pmlandwehr** [comment]
@jakirkham doing one more Circle CI attempt, then merging.

**Comment by jakirkham** [comment]
Have you tried merging with upstream `master` yet?

**Comment by pmlandwehr** [comment]
good note.

File: recipes/oslo.serialization/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    - six >=1.9.0
+    - msgpack-python >=0.4.0
+    - oslo.utils >=3.11.0
```
Comment (jakirkham) [inline recipes/oslo.serialization/meta.yaml]: I think the issue is we are missing this dependency for some Python versions on Linux.

xref: https://github.com/conda-forge/oslo.utils-feedstock/issues/1

File: recipes/oslo.serialization/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  build:
+    - python
+    - setuptools
+    - pbr >=1.8
+
+  run:
+    - python
+    - pbr >=1.6
+    - six >=1.9.0
+    - msgpack-python >=0.4.0
```
Comment (jakirkham) [inline recipes/oslo.serialization/meta.yaml]: Do we have a PR here at staged-recipes for this one? If not, could we please add one?

**Pull request closed without merging**